### ISSUE TITLE
several bugs found.

### ISSUE THREAD

**Issue by VaGNaroK** [linuxmint/mint20-beta#25]
Test linux mint cinnamon beta x64:

*area not translated in the brazilian portuguese application manager:
https://postimg.cc/2VfSRWkc

*authorization to use synaptic in English and not in Portuguese even with the system added translations. (this has always been a problem):
https://postimg.cc/gLydkMzM

*dependency bug when installing codec causing the system report window to crash:
https://postimg.cc/4YyCr492
https://postimg.cc/QBK2d6Sp

*Install language window not translated into Portuguese even with the installed languages:
https://postimg.cc/hXXDV1pG

*text in xed not translated even though it has translation in the launchpad:
https://postimg.cc/Wtss5vXC

*root permission on the English updater even with language pack installed:
https://postimg.cc/sQmdkMw8

*untranslated text in the system report even though it is already translated in the launchpad:
https://postimg.cc/f3vrL6mS

*translation missing warpinator, even with translation on launchpad:
https://postimg.cc/sB3Ttwqj

*in monitor settings, translations not applied even though it's right on the launchpad:
https://postimg.cc/8j8B2Zf2

So far I only found these.

**Comment by enigma131** [comment]
*dependency bug when installing codec causing the system report window to crash: duplicate of #11

**Comment by kongreddgrass** [comment]
If i can read: One issue per bug ;)

**Comment by NikoKrause** [comment]
I'll try to break this down:

- [x] **Already fixed**
area not translated in the brazilian portuguese application manager:
https://postimg.cc/2VfSRWkc

- [x] **Can't try in live mode:**
 authorization to use synaptic in English and not in Portuguese even with the system added translations. (this has always been a problem):
https://postimg.cc/gLydkMzM

- [x] **Known bug:**
dependency bug when installing codec causing the system report window to crash:
https://postimg.cc/4YyCr492
https://postimg.cc/QBK2d6Sp

- [ ] **Can't try in live mode**
Install language window not translated into Portuguese even with the installed languages:
https://postimg.cc/hXXDV1pG

- [x] **Created PR https://github.com/linuxmint/xed/pull/385**
text in xed not translated even though it has translation in the launchpad:
https://postimg.cc/Wtss5vXC

- [ ] **Can't try in live mode**
root permission on the English updater even with language pack installed:
https://postimg.cc/sQmdkMw8

- [x] **Translation for mintreport haven't been updated yet, should work though, when it's in launchpad**
untranslated text in the system report even though it is already translated in the launchpad:
https://postimg.cc/f3vrL6mS

- [x] **Translation for warpinator haven't been updated yet, should work though, when it's in launchpad**
 translation missing warpinator, even with translation on launchpad:
https://postimg.cc/sB3Ttwqj

- [x] **Already fixed**
in monitor settings, translations not applied even though it's right on the launchpad:
https://postimg.cc/8j8B2Zf2

**Comment by clefebvre** [comment]
Synaptic fixed

**Comment by clefebvre** [comment]
I don't really care about the live session to be honest.

Once the language is set to pt_BR and a logout is done, everything's in portuguese now.

**Comment by NikoKrause** [comment]
I couldn't check the strings in live mode, because I don't get asked for a password

**Comment by xenopeek** [comment]
> I couldn't check the strings in live mode, because I don't get asked for a password

You can create a new admin user in live mode, log out and log in to that for testing such.
